module/Endjin.ZeroFailed.Build.psd1

```diff
@@ -0,0 +1,141 @@
+# <copyright file="Endjin.ZeroFailed.Build.psd1" company="Endjin Limited">

```

Comment:
Is there a way to describe the tooling dependencies for this module? `gh`, `az` etc?